cigaradvisor/blocks/article-list/article-list.js

```diff
@@ -29,10 +29,10 @@ export async function renderPage(wrapper, articles, limit) {
     const articleTeaser = document.createElement('div');
     articleTeaser.classList.add('article-teaser');
     articleTeaser.classList.add('block');
-    if (typeof article.author === 'string') {
-      article.author = await fetchAuthorInfo(article.author);
+    if (article.author) {
+      article.author = typeof article.author === 'string' ? await fetchAuthorInfo(article.author) : '';
     }
```

Comment:
This will break the author - if it is already populated with an object, then it replaces that object with an empty string.

Reply:
for some article article.author was null.